src/s11/handlers/s11_msg_delegator.c

```diff
@@ -170,6 +170,9 @@ handle_s11_message(void *message)
 
 	bool rc = GtpV2Stack_decodeMessageHeader(gtpStack_gp, &msgHeader, msgBuf_p);
 
+	if(rc == false)
+		log_msg(LOG_ERROR,"Failed to decode the GTP Message\n");
```

Comment:
Put the hexdump in the log message. THis will help in knowing what failed.

Reply:
Done.